pkg/scheduler/framework/interface.go

```diff
@@ -478,7 +478,7 @@ type Framework interface {
 	// preemption, we may pass a copy of the original nodeInfo object that has some pods
 	// removed from it to evaluate the possibility of preempting them to
 	// schedule the target pod.
-	RunFilterPlugins(ctx context.Context, state *CycleState, pod *v1.Pod, nodeInfo *NodeInfo) PluginToStatus
+	//RunFilterPlugins(ctx context.Context, state *CycleState, pod *v1.Pod, nodeInfo *NodeInfo) PluginToStatus
```

Comment:
Can you move removal of the methods into a separate PR? Building your changes locally without changing the `Framework` interface, I don't see any compilation errors.

Reply:
Are these comments intentional?

Reply:
I see the following error in my GoLand IDE. While this is not causing compilation issues, I see the following:

```
Duplicate method 'RunFilterPlugins'
Duplicate method 'RunPreFilterExtensionAddPod'
Duplicate method 'RunPreFilterExtensionRemovePod'
Duplicate method 'RunPreScorePlugins'
Duplicate method 'RunScorePlugins'
```

coming from ```Handle -> PluginsRunner```
I was initially thinking it was caused by my changes but seems master is also having the same problem. Unrelated but why to have those methods implicitly and explicitly within embedded interfaces and Framework interface?

Reply:
Ah... it seems like a leftover from a refactoring. I'll open a separate PR for that.